plotly/grid_objs/grid_objs.py

```diff
@@ -187,3 +254,17 @@ def get_column(self, column_name):
         for column in self._columns:
             if column.name == column_name:
                 return column
+
+    def get_fid_uid(self, column_name):
+        """
+        Return uid of given column name in the grid by column name.
+
+        Returns an empty string if either the column name does not exist in
+        the grid or if the id of the specified column has the empty string id.
+        """
+        uid = ''
+        for column in self._columns:
+            if column.name == column_name:
+                uid = column.id
+                break
+        return uid
```

Comment:
grids are instantiated with `id`s so I don't think that's a risk here. For now, I think it's fine to have it just return the `id` of that column, whatever it happens to be.